processor/tailsamplingprocessor/config.go

```diff
@@ -159,9 +159,9 @@ type NumericAttributeCfg struct {
 	// Tag that the filter is going to be matching against.
 	Key string `mapstructure:"key"`
 	// MinValue is the minimum value of the attribute to be considered a match.
-	MinValue int64 `mapstructure:"min_value"`
+	MinValue *int64 `mapstructure:"min_value"`
```

Comment:
we use configoptional.Optional for optional values. Pointer values cause much chagrin in some situations.